command_test.go

```diff
@@ -3068,26 +3068,29 @@ func TestPersistentFlag(t *testing.T) {
 }
 
 func TestFlagDuplicates(t *testing.T) {
-	cmd := &Command{
-		Flags: []Flag{
-			&StringFlag{
-				Name:     "sflag",
-				OnlyOnce: true,
-			},
-			&IntSliceFlag{
-				Name: "isflag",
-			},
-			&FloatSliceFlag{
-				Name:     "fsflag",
-				OnlyOnce: true,
+
+	cmd_get := func() *Command {
```

Comment:
If you need a fresh one for each test case, maybe define inside the `t.Run(...)` scope?

Reply:
Yeah makes sense. Done